changelog/unrelreased/1863--broker-writer.md

```diff
@@ -0,0 +1,4 @@
+The new `broker` writer plugin enables exporting query results directly into a
```

Comment:
Please clarify that this is not actually a new plugin: The broker plugin now is both a reader and a writer plugin.